covid19_sfbayarea/data/san_mateo/__init__.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from datetime import datetime
+from dateutil import tz
+from typing import Any, Dict
+
+from .cases_by_age import CasesByAge
+from .cases_by_ethnicity import CasesByEthnicity
+from .cases_by_gender import CasesByGender
+
+from .meta import Meta
+
+from .deaths_by_age import DeathsByAge
+from .deaths_by_ethnicity import DeathsByEthnicity
+from .deaths_by_gender import DeathsByGender
+
+from .time_series_cases import TimeSeriesCases
+from .time_series_tests import TimeSeriesTests
+from ..utils import get_data_model
+
+LANDING_PAGE = 'https://www.smchealth.org/post/san-mateo-county-covid-19-data-1'
+
+def get_county() -> Dict:
+    out = get_data_model()
+    out.update(fetch_data())
+    return out
+
+def fetch_data() -> Dict:
+    data = {
+        'name': 'San Mateo County',
+        'source_url': LANDING_PAGE,
+        'meta_from_source': Meta().get_data(),
+        'meta_from_baypd': """
+            See power_bi_scraper.py for methods.
+            San Mateo does not provide a timestamp for their last dataset update,
+            so BayPD uses midnight of the latest day in the cases timeseries as a proxy.
```

Comment:
San Mateo does not provide a deaths timeseries. In lieu of a timeseries BayPD provides cumulative deaths for the date of the last dataset update.